source/StigData/Processed/WindowsServer-2016-DC-2.3.xml

```diff
@@ -3508,6 +3507,7 @@ Inherited from - "None" for all
 Principal - Access - Applies to
 SYSTEM - Full Control - This key and subkeys
 Administrators - Special - This key and subkeys
+Server Operators â€“ Read â€“ This Key and subkeys    (Domain controllers only)
```

Comment:
We need to make sure Server 2016 Domain Controller has this new User Right Assignment - Server Operators - Read - This Key and Subkeys..